src/navigator.js

```diff
@@ -271,6 +273,50 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
             }
         }
     },
+    /**
+      /* Flip navigator element
+      */
+    toogleFlip: function() {
+      this.flipped = !this.flipped;
+      if (this.viewer.flipped){
+        this.displayRegion.style.webkitTransform = "scale(-1,1)";
```

Comment:
Thank you for being thorough here! Since there are so many of these, it seems worth making a helper function to encapsulate the 5 transform variations.

Reply:
Sure, I'll make a change on the next commit.